### PR TITLE
Add Parquet symbols nuget package.

### LINKED ISSUE #144: Microsoft.ML.Parquet doesn't have a symbols package
### System information

- **OS version/distro**: all
- **.NET Version (eg., dotnet --info)**:  N/A

### Issue

- **What did you do?**
Try to load symbols for the Microsoft.ML.Parquet package
- **What happened?**
There are no symbols uploaded to the symbols server
- **What did you expect?**
I expected the symbols to be on the symbols server, just like Microsoft.ML.

### Source code / logs

We don't have a "symbols.nupkgproj" here:  https://github.com/dotnet/machinelearning/tree/master/pkg/Microsoft.ML.Parquet
But we do for https://github.com/dotnet/machinelearning/tree/master/pkg/Microsoft.ML.

### PR DESCRIPTION
**Pull request by eerhardt**
Add symbols package for ML.Parquet package.
Put common NuGet package logic in props file.

Fix #144

### CODE REVIEW

File: pkg/Directory.Build.props [hunk 1]
```diff
@@ -24,6 +27,17 @@
     <PackageReleaseNotes>https://github.com/dotnet/machinelearning/tree/master/Documentation/release-notes</PackageReleaseNotes>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\lib\**\*" Pack="true" PackagePath="lib" />
+    <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\runtimes\**\*" Pack="true" PackagePath="runtimes" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsSymbolsPackage)' != 'true'">
```
Comment (Anipik) [inline pkg/Directory.Build.props]: there will be some symbols with the extension .dll, this wont remove them

Comment (eerhardt) [reply pkg/Directory.Build.props]: what symbols have an extension of `.dll`?

**Comment by eerhardt** [comment]
@dotnet-bot test this please

File: pkg/Directory.Build.props (hunk 1 above)
Comment (Anipik) [reply pkg/Directory.Build.props]: Sorry, my bad I got confused.

**Pull request merged**